File: 389-find-the-difference/389-find-the-difference.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        
        
        s_list=list()
        t_list=list()
        
        for i in s:
            s_list.append(i)
            
        for i in t:
            t_list.append(i)
            
        s_list.sort()
        t_list.sort()
        
        ssorted=""
        tsorted=""
        
        for i in s_list:
            ssorted+=i
        for j in t_list:
            tsorted+=j
            
        for i in range(0,len(ssorted)):
            if(ssorted[i]!=tsorted[i]):
                return tsorted[i]
        
        return tsorted[-1]
            
        

File: 389-find-the-difference/389-find-the-difference.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        
        int sums=0;
        int sumt=0;
        for(int i=0;i<s.size();i++)
        {
            sums=sums+s[i];
        }
        for(int i=0;i<t.size();i++)
        {
            sumt=sumt+t[i];
        }
        return char(sumt-sums);
        
        
        
    }
};